packages/cascara/src/ui/Title/Title.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Boundaries } from '../../system-components';
+import { Flex } from '../../atoms';
+
+import styles from './Title.module.scss';
+import { PROP_TYPES } from './__globals';
+
+const PageTitle = ({ titlePost, titlePre, titleSub, title }) => {
+  const titleTag = title && (
+    <h3 className={styles.Title}>
```

Comment:
Manu, this is not something we have done in the other app where this came from... but this is a good candidate for using `Role` and `as` on the title. It means we also need to expose a new `titleAs` prop with a default of `h3`.

Reply:
@brianespinosa This is much better now, thanks for cathing this opportunity 👍